Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Use multi-stage build
+FROM golang:1.15 as builder
```

Comment:
We should use a ubuntu image - probably have to install go on our own too.  Can reference the rosetta-etherium image for installing go from source in a ubuntu image.

Also, should create  a `/app` directory to clone the repositories into, then create a second stage and copy the binaries over.